tsconfig.json

```diff
@@ -9,8 +9,8 @@
       "ES2015", "ES2016", "ES2017", "ES2018",
       "ES2019", "ES2020","ES2021", "DOM"
     ],
-    "module": "commonjs",
-    "moduleResolution": "node",
+    "module": "nodenext",
```

Comment:
For details: https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c#how-can-i-make-my-typescript-project-output-esm